### DISCUSSION TITLE
How to use azure-sdk-for-go v1.0.0 and latest version of go-mssqldb?

### DISCUSSION THREAD

**Discussion by bramwesselo** [microsoft/go-mssqldb#29] [Q&A] [answered]
I'd like to use the azure-sdk-for-go/sdk/storage/azblob (v0.4.0 or above) AND github.com/microsoft/go-mssqldb/azuread. 
(to query data from a database with AAD-logon and upload a csv/parquet-file to a storage account).

However, package ...azblob needs (among others) package github.com/Azure/azure-sdk-for-go/sdk/azidentity **v0.14.0** (or v1.0.0) , while ...go-mssqldb/azuread needs ...azidentity **v0.11.0** (and doesn't work with a later version).

Is there a way to make this work?

**Comment by bramwesselo** [comment, marked as answer]
@shueybubbles Do you think I should open a new Issue (in stead of waiting for an answer)?

**Reply by stuartpa** [reply]
Do we just need to get #13 merged? (from the commits in that PR it seems to have wavered between 0.13 and 0.14.  Can we get it merged as 0.14?

**Reply by shueybubbles** [reply]
@gambtho Can you merge #13 with v0.14.0 to unblock this?

**Reply by bramwesselo** [reply]
Great; this solved the issue I had!
Thanks
